api.py

```diff
@@ -0,0 +1,18 @@
+ 
```

Comment:
Please find the requried dependencies and fix them.

['flask', '    google-cloud-storage']